openhtf/exe/test_state.py

```diff
@@ -87,6 +87,19 @@ def _asdict(self):
         'pending_phases': [(phase.func.__name__, phase.func.__doc__)
                            for phase in self.pending_phases]}
 
+  def GetRunningPhaseName(self):
```

Comment:
ah, yeah it was modeled after `self.current_phase_record`, but a more accurate name is probably a good idea, done!